Index: build/resources/main/application.properties
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>\r\n# Database connection settings\r\nspring.datasource.url=jdbc:mysql://localhost:3306/techjobs_auth\r\nspring.datasource.username=techjobs_auth\r\nspring.datasource.password=auth\r\n\r\n# Specify the DBMS\r\nspring.jpa.database = MYSQL\r\n\r\n# Show or not log for each sql query\r\nspring.jpa.show-sql = false\r\nspring.main.allow-bean-definition-overriding=true\r\n\r\n# Hibernate ddl auto (create, create-drop, update)\r\nspring.jpa.hibernate.ddl-auto = update\r\n\r\n# Use spring.jpa.properties.* for Hibernate native properties (the prefix is\r\n# stripped before adding them to the entity manager)\r\nspring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5Dialect\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/build/resources/main/application.properties b/build/resources/main/application.properties
--- a/build/resources/main/application.properties	(revision 1f7b3a96510901cffcc0fac7ffb09a5477e354de)
+++ b/build/resources/main/application.properties	(date 1629648545717)
@@ -8,7 +8,7 @@
 spring.jpa.database = MYSQL
 
 # Show or not log for each sql query
-spring.jpa.show-sql = false
+spring.jpa.show-sql = true
 spring.main.allow-bean-definition-overriding=true
 
 # Hibernate ddl auto (create, create-drop, update)
